library/alloctests/testing/macros.rs

```diff
@@ -1,13 +1,33 @@
 macro_rules! struct_with_counted_drop {
```

Comment:
Not necessarily something to do here, but could vecdeque just use these too?

Reply:
I'm not sure what you mean here.